maven-embedder/src/main/java/org/apache/maven/cli/ExtensionConfigurationModule.java

```diff
@@ -27,18 +27,18 @@
 import org.apache.maven.internal.xml.XmlPlexusConfiguration;
 import org.apache.maven.model.v4.MavenTransformer;
 import org.codehaus.plexus.configuration.PlexusConfiguration;
+import org.codehaus.plexus.interpolation.AbstractValueSource;
 import org.codehaus.plexus.interpolation.InterpolationException;
-import org.codehaus.plexus.interpolation.PropertiesBasedValueSource;
 import org.codehaus.plexus.interpolation.StringSearchInterpolator;
 
 class ExtensionConfigurationModule implements Module {
```

Comment:
Could this be a public class I think m2e will want to reuse that class at some point in time....